pkg/constants/metadata.go

```diff
@@ -23,6 +23,9 @@ const (
 	// DevWorkspaceNameLabel is label key to store workspace name
 	DevWorkspaceNameLabel = "controller.devfile.io/devworkspace_name"
 
+	// DevWorkspaceAutomountLabel is label key to store if a configmap or secret she should be automounted to the devworkspace
+	DevWorkspaceAutomountLabel = "controller.devfile.io/automount-to-workspace"
```

Comment:
hello, I was wondering if we need the `auto` prefix because I don't see why it can't be automatically mounted if I only specify "controller.devfile.io/mount-to-workspace" ( auto = the job of the controller anyway)

for example in devfiles I have `mountSources : true|false`

Reply:
@benoitf makes sense, I've changed the label to the following: 
```
DevWorkspaceMountLabel = "controller.devfile.io/mount-to-workspace"
```